### PR TITLE
Pbd monitoring

### PR DESCRIPTION
**Pull request by jklukas**
See https://mozilla-hub.atlassian.net/browse/DSRE-174

Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title)
- [ ] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
- [ ] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated

### CODE REVIEW

File: tests/sql/moz-fx-data-shared-prod/telemetry_derived/urlbar_clients_daily_v1/test_overactive_filter/telemetry.clients_daily.yaml [hunk 1]
```diff
@@ -41,4 +41,4 @@
     - key: "0"
       value: 6
     - key: "1"
-      value: 10001 # 1 over the threshold
+      value: 10001  # 1 over the threshold
```
Comment (jklukas) [inline tests/sql/moz-fx-data-shared-prod/telemetry_derived/urlbar_clients_daily_v1/test_overactive_filter/telemetry.clients_daily.yaml:44]: Fixing a random additional yamllint warning

**Comment by jklukas** [comment]
Remaining CI failures are due to the `_all` view depending on the other views that don't yet exist.

File: sql/moz-fx-data-shared-prod/monitoring_derived/deletion_request_volume_v1/query.sql [hunk 2]
```diff
@@ -1,13 +1,13 @@
 SELECT
   DATE(submission_timestamp) AS submission_date,
-  REGEXP_EXTRACT(_TABLE_SUFFIX, "(?:stub_)?(?:[^_]+_)(.*)__") AS dataset_id,
+  REPLACE(metadata.document_namespace, '-', '_') AS dataset_id,
   COUNT(*) AS num_rows,
 FROM
-  `payload_bytes_decoded.*`
+  `payload_bytes_decoded_all`
```
Comment (whd) [inline sql/moz-fx-data-shared-prod/monitoring_derived/deletion_request_volume_v1/query.sql:6]: I was wondering why this works without table is unqualified.

File: sql/moz-fx-data-shared-prod/monitoring_derived/structured_distinct_docids_v1/query.py [hunk 3]
```diff
@@ -61,7 +60,8 @@
   submission_date
 """
 
-EXCLUDED_NAMESPACES = {"xfocsp_error_report"}  # restricted access
+# restricted access
+EXCLUDED_NAMESPACES = {"xfocsp_error_report", "contextual_services"}
```
Comment (whd) [inline sql/moz-fx-data-shared-prod/monitoring_derived/structured_distinct_docids_v1/query.py:64]: Should this matter for monitoring queries on authorized views? One of these is actually a doctype under the telemetry namespace so maybe was ignored or was supposed to be `regrets-reporter`.

File: bigquery_etl/dryrun.py [hunk 4]
```diff
@@ -68,6 +68,9 @@
     "sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_amplitude_user_ids_v1/init.sql",  # noqa E501
     "sql/moz-fx-data-shared-prod/regrets_reporter/regrets_reporter_update/view.sql",
     "sql/moz-fx-data-shared-prod/revenue_derived/client_ltv_v1/query.sql",
+    "sql/moz-fx-data-shared-prod/monitoring/payload_bytes_decoded_structured/view.sql",
```
Comment (whd) [inline bigquery_etl/dryrun.py:71]: does `all` need to be here?

**Review by whd — COMMENTED** [review]
CI still looks sad

File: sql/moz-fx-data-shared-prod/monitoring_derived/deletion_request_volume_v1/query.sql (hunk 2 above)
Comment (jklukas) [inline sql/moz-fx-data-shared-prod/monitoring_derived/deletion_request_volume_v1/query.sql:6]:
```suggestion
  monitoring.payload_bytes_decoded_all
```

File: bigquery_etl/dryrun.py (hunk 4 above)
Comment (jklukas) [reply bigquery_etl/dryrun.py:71]: No, it should be dry-runnable once the authorized views it references are fully published.

File: sql/moz-fx-data-shared-prod/monitoring_derived/structured_distinct_docids_v1/query.py (hunk 3 above)
Comment (jklukas) [reply sql/moz-fx-data-shared-prod/monitoring_derived/structured_distinct_docids_v1/query.py:64]: I think Airflow will have necessary privs to access all of these; not sure why this was included originally. Maybe this is totally unnecessary now.

File: sql/moz-fx-data-shared-prod/monitoring_derived/deletion_request_volume_v1/query.sql (hunk 2 above)
Comment (jklukas) [reply sql/moz-fx-data-shared-prod/monitoring_derived/deletion_request_volume_v1/query.sql:6]: It needs to be qualified.

File: sql/moz-fx-data-shared-prod/monitoring_derived/structured_distinct_docids_v1/query.py (hunk 3 above)
Comment (jklukas) [inline sql/moz-fx-data-shared-prod/monitoring_derived/structured_distinct_docids_v1/query.py:64]:
```suggestion
EXCLUDED_NAMESPACES = {"xfocsp_error_report"}  # restricted access
```

**Review by whd — APPROVED** [review]
Didn't see the comment about CI. PBD isn't restricted but landing this in reverse order and applying authorizations afterwards should work. I'll have the infra PBD PR up later today.

**Pull request merged**